CMS-Enterprise/easi-app
# [EASI-4555] GRB participants table

# EASI-4555

[Figma](https://www.figma.com/design/G0UN35V7SY0g9nkTIESxsY/EASi-GRB-Experience?node-id=2597-48335&t=IymgRhaAzWpvG86B-1)

## Description
- Added participants table to GRB review tab
- Unit tests

**Note**
- Query is not using codegen - all GRB queries/mutations will be updated in followup NOREF once #2724 is merged
- ~~Edit/Remove action buttons are not functional. This will be fixed after #2724 is merged.~~


## How to test this change
- Log in as `ABCD` with `EASI_GOVTEAM` job code
- Go to /governance-review-team/5af245bc-fc54-4677-bab1-1b3e798bb43c/grb-review
- Table should match Figma
  - Action buttons column should render
- Log in as `ABCD` with `EASI_USER` job code
- Go to /governance-review-board/5af245bc-fc54-4677-bab1-1b3e798bb43c/grb-review
- Table should match Figma
  - Action buttons column should NOT render

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
